### ISSUE TITLE
respect build instructions like // +build go1.9

### ISSUE THREAD

**Issue by Dexus** [360EntSecGroup-Skylar/goreporter#25]
Please dont run into crashes while disrespect `// +build go1.9` comments...
Cant check now my own code, because of use 1.8.3 but the tools check also go1.9 code...

**Comment by wgliang** [comment]
go1.9 is not a stable version,so I did not test whether it was supported.

**Comment by wgliang** [comment]
go1.9beta2 is not a stable version and I did not test whether it was supported. You can post your error and I will test it!

**Comment by Dexus** [comment]
github.com/go-siris/siris

**Comment by wgliang** [comment]
I test it and the error message is not goreporter does not skip the vendor directory,but required in 
 these packages in the unit test , so you must ensure that the project is runable.

**Comment by wgliang** [comment]
I have tested siris. The error message is not goreporter does not skip the vendor directory,but siris required these packages in the unit test , so you must ensure that the project is runable.

**Comment by Dexus** [comment]
yeah, i have add now all the sources but its fails with the abdove error.
